packages/core/src/internal/channel.js

```diff
@@ -183,14 +194,20 @@ export function multicastChannel() {
       }
 
       const takers = (currentTakers = nextTakers)
-      takers.forEach(taker => {
+
+      for (let i = 0, len = takers.length; i < len; i++) {
```

Comment:
this is a hot path, so a for loop should be better here, perf-wise

Reply:
ok, then let's add a comment that the for-loop is intentional